### PR TITLE
use Optional instead of Variant[Undef...

### PR DESCRIPTION
**Pull request by TheMeier**
align indentation

### CODE REVIEW

File: manifests/nginx_vts_exporter.pp [hunk 1]
```diff
@@ -72,29 +72,29 @@
...
-  Boolean $restart_on_change          = true,
-  Boolean $service_enable             = true,
-  String $service_ensure              = 'running',
-  String $user                        = $::prometheus::params::nginx_vts_exporter_user,
-  String $version                     = $::prometheus::params::nginx_vts_exporter_version,
+  String            $arch               = $::prometheus::params::arch,
+  String            $bin_dir            = $::prometheus::params::bin_dir,
+  String            $nginx_scrape_uri   = $::prometheus::params::nginx_vts_exporter_nginx_scrape_uri,
+  String            $download_extension = $::prometheus::params::nginx_vts_exporter_download_extension,
+  Optional[String] $download_url        = undef,
```
Comment (wyardley) [inline manifests/nginx_vts_exporter.pp]: This looks good to me, but if you're going to line up the params, I think you need one more space here? Or, alternately, do what most modules do, and don't align the second column (though to me, this actually is easier to read).

I think if it's >= Puppet 4, you can also go back to `$prometheus` vs `$::prometheus` if you like (the former is actually preferred now).

**Review by wyardley — APPROVED** [review]
Other than the formatting, this looks good to me... do you want to fix the spacing and squash?

File: manifests/nginx_vts_exporter.pp (hunk 1 above)
Comment (TheMeier) [reply manifests/nginx_vts_exporter.pp]: thanks for the input. fixed

**Comment by alexjfisher** [comment]
@TheMeier Could you squash those 2 commits?  Thanks.

**Comment by TheMeier** [comment]
squashed

**Comment by alexjfisher** [comment]
@TheMeier Thanks!

**Pull request merged**